lib/models/graph-object.js

```diff
@@ -43,6 +43,38 @@ function GraphModelFactory (Model, Constants, configuration) {
             active: function() {
                 var obj = this.toObject();
                 return Constants.Task.ActiveStates.indexOf(obj._status) > -1;
+            },
+            toJSON: function () {
```

Comment:
@yyscamper so Felix, the toJSON is an override of the base function in waterline for toJSON, so this is actually invoked automatically when a query is made (i've tested this)... however... I cannot have my cake and eat it too... I am unable to make unit tests using an override function like this, because of scope issues... I'll have to revert to making a class function and calling it in on-http like you suggest.